packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -120,6 +120,7 @@ export class DebugSessionManager {
             const sessionId = await this.debugService.create(resolved.configuration);
             return this.doStart(sessionId, resolved);
         } catch (e) {
+            this.messageService.error(e);
```

Comment:
if there are programming errors, they should be only logged. We should ignore `NotFound` errors, they are expected and should not be looged.

If there are user errors from the debug adapter, then the debug adapter contribution should use `MessageService` to provide meaningful errors, like java and node contributions already do.

Reply:
Even if it's a programming error, shouldn't we at least show a little something, like "Woops, internal error, please check the logs"?  Otherwise, the user clicks the start button, and nothing happens at all...

Reply:
ok